.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ env:
   APP_NAME: "nautobot-app-dev-example"
 
 jobs:
-  black:
+  formatting:
```

Comment:
Wouldn't be `ruff-format` a better name for this job? Same `ruff-lint` for the other job.

I was thinking about making separate invoke tasks for these, as both ruff actions are actually different ruff commands with different arguments.

- `invoke ruff-format`
- `invoke ruff-lint`

Opinions?

Reply:
In nautobot core we have a [single `invoke ruff` task](https://github.com/nautobot/nautobot/blob/develop/tasks.py#L586-L616); I'd be in favor of following that pattern, especially as the format/lint split in ruff is fairly arbitrary (for example, applying isort rules to imports is part of linting, not part of formatting??).